web/src/app/projects/view/view.component.ts

```diff
@@ -58,10 +64,23 @@ export class ViewProjectComponent implements OnInit, OnDestroy {
 
   // This function delete the project
   delete(): void {
-    this.projectSubscription.unsubscribe();
-    this.deleteSubscription = this.projectService
-      .delete(this.project.uid)
-      .subscribe(() => this.router.navigate(['/projects']));
+    let dialogConfig: MatDialogConfig = new MatDialogConfig();
+    dialogConfig = {
+      width: '500px',
+      data: {
+        title: 'Delete Project',
+        content: 'Are you sure you want to delete?',
+      },
+    };
+    this.dialogRef = this.dialog.open(DialogConfirmationComponent, dialogConfig);
+    this.dialogRef.afterClosed().subscribe((result: boolean) => {
```

Comment:
Easier to read if the `subscribe` is on a new line

Reply:
fixed